CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+Version 8.3.2
+~~~~~~~~~~~~~
+
+Bug fix release:
+
+All changes:
+https://github.com/Open-MSS/MSS/milestone/100
```

Comment:
Should this include `?closed=1`, as in many previous changelog entries? Also, the milestone is not exhaustive, i.e. the issue/PR for the pint pin is not listed there.

Reply:
yes, but that was a failure which was not seen on the release creation. this is a merge. we don't do changes in a merge.

Reply:
that can be fixed when we do a change on docs, or release later.

Reply:
OK, and I just realized that the pint pin change is not part of 8.3.2 but will be in 8.3.3 instead.

Reply:
yes, on conda-forge it is already in the build -1, this is to update our code base.